elastic/kibana
# [Infra UI] Add date picker to asset details

closes: https://github.com/elastic/kibana/issues/160378

## Summary

This PR adds a date-picker to the Asset Details (full page and flyout)

**Flyout**
<img width="1435" alt="image" src="https://github.com/elastic/kibana/assets/2767137/a1adb3b0-b3d5-4ca3-9076-6dafd44eebbc">


**Full page view**
<img width="1422" alt="image" src="https://github.com/elastic/kibana/assets/2767137/ad14764b-fdc2-4773-b8d0-7f2e271d89a5">

**Setting a date range**

https://github.com/elastic/kibana/assets/2767137/4c41d9ee-6c18-45af-9bda-3cae6f372b7b

**Brush event filter**

https://github.com/elastic/kibana/assets/2767137/5830dcd7-7d66-45ed-a463-15bb79607f2a

### Bonus

The uptime link was removed since the UI won't work.

### How to test

- Start a local Kibana instance
- Start metricbeat with system module enabled
- Navigate to `Infrastructure` > `Hosts`
  - Click on a host name and check if the new Node Details page opened
      - Check if the date range is set with the same date range from the Host Detail view
- Navigate to `Infrastructure` > `Hosts`
   -  Open the flyout
   -  Click on Open as page and check if the new Node Details page opened with the same date range from the Host Detail view
- Navigate to Infrastructure > Inventory
      - With Show: Host selected, click on a waffle item to open the flyout, click on Open as page and check if the new Node Details page opened with the same date range from the Inventory UI

Validate if the date picker value is applied to all tabs that have the component (Overview, Metadata, Processes, Logs, Anomalies)
Validate if the date range changes when selecting a time range in the charts (in the flyout, the date range from the Hosts View must not change)
